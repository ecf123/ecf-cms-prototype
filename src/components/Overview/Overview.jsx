import "./Overview.scss";

const Overview = ({ header, text }) => {
  return (
    <div className="overview">
      <h2 className="overview__header">{header}</h2>
      <p className="overview__text">{text}</p>
    </div>
  );
};

export default Overview;
