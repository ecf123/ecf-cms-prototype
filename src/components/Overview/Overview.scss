@use "../../styles/_variables" as *;
@use "../../styles/_typography" as *;

.overview {

    &__header {
        color: $color-blue-navy-dark;
        font-family: $font-secondary;
    }

    &__text {
        color: $color-teal;
        font-family: $font-secondary;
    }
}