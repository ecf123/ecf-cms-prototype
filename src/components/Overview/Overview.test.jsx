import { customRender } from "../../utils/testUtils";
import {render, screen} from "@testing-library/react";
import Overview from "./Overview"


it("renders the Overview", () => {
  const { container } = customRender(<Overview />);
  expect(container).toMatchSnapshot();
});

it("should render the header on the screen", () => {
  render(<Overview header="Overview"/>)
  const header = screen.getByText("Overview");

  expect(header).toBeInTheDocument();
})

it("should render the text on the screen", () => {
  render(<Overview text="text"/>)
  const text = screen.getByText("text");

  expect(text).toBeInTheDocument();
})

