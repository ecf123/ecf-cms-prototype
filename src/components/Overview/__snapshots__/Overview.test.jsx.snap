// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Overview 1`] = `
<div>
  <div
    class="overview"
  >
    <h2
      class="overview__header"
    />
    <p
      class="overview__text"
    />
  </div>
</div>
`;
